<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageprocessClass</class>
 <widget class="QMainWindow" name="imageprocessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imageprocess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>391</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>载入图像</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>369</width>
          <height>317</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>391</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>处理显示</string>
    </property>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>319</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>369</width>
        <height>317</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>图像处理</string>
    </property>
    <widget class="QMenu" name="menu_edge">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>边缘检测</string>
     </property>
     <addaction name="actionCanny"/>
     <addaction name="actionSobel"/>
     <addaction name="actionLaplacian"/>
    </widget>
    <widget class="QMenu" name="menu_histogram">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>直方图</string>
     </property>
     <addaction name="actionR"/>
     <addaction name="actionG"/>
     <addaction name="actionB"/>
     <addaction name="actionGry"/>
    </widget>
    <addaction name="menu_edge"/>
    <addaction name="separator"/>
    <addaction name="actionErode"/>
    <addaction name="actionDilate"/>
    <addaction name="separator"/>
    <addaction name="menu_histogram"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>滤波</string>
    </property>
    <widget class="QMenu" name="menu_line">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>线性滤波</string>
     </property>
     <addaction name="actionBox"/>
     <addaction name="actionMeans"/>
     <addaction name="actionGuass"/>
    </widget>
    <widget class="QMenu" name="menu_nonline">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>非线性滤波</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionMidvalue"/>
     <addaction name="actionDoubledge"/>
    </widget>
    <addaction name="menu_line"/>
    <addaction name="menu_nonline"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>人脸检测</string>
    </property>
    <addaction name="actionFacedetect"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionErode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>腐蚀</string>
   </property>
  </action>
  <action name="actionDilate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>膨胀</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionLaplacian">
   <property name="text">
    <string>Laplacian</string>
   </property>
  </action>
  <action name="actionR">
   <property name="text">
    <string>R通道</string>
   </property>
  </action>
  <action name="actionG">
   <property name="text">
    <string>G通道</string>
   </property>
  </action>
  <action name="actionB">
   <property name="text">
    <string>B通道</string>
   </property>
  </action>
  <action name="actionGry">
   <property name="text">
    <string>灰度直方图</string>
   </property>
  </action>
  <action name="actionBox">
   <property name="text">
    <string>方框滤波</string>
   </property>
  </action>
  <action name="actionMeans">
   <property name="text">
    <string>均值滤波</string>
   </property>
  </action>
  <action name="actionGuass">
   <property name="text">
    <string>高斯滤波</string>
   </property>
  </action>
  <action name="actionMidvalue">
   <property name="text">
    <string>中值滤波</string>
   </property>
  </action>
  <action name="actionDoubledge">
   <property name="text">
    <string>双边滤波</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="imageprocess.qrc">
     <normaloff>:/imageprocess/Resources/reset (2).png</normaloff>:/imageprocess/Resources/reset (2).png</iconset>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </action>
  <action name="actionFacedetect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>人脸检测</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imageprocess.qrc"/>
 </resources>
 <connections/>
</ui>
